tests/test_reserver.py

```diff
@@ -8,6 +8,12 @@ def test_package_exists():
     uploader = Uploader(test_pypi_token, test_pypi= True)
     assert uploader.upload_to_pypi("numpy") == False
 
+def test_batch_packages_names():
+    # test batch of package names
+    uploader = Uploader(test_pypi_token, test_pypi= True)
+    uploader.batch_upload_to_pypi("numpy", "scikit-learn")
+    assert True == True 
```

Comment:
If you accept my comment on returning `True` for `upload_to_pypi`, you can assert the output of that function here.